StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1306UnitTests.cs

```diff
@@ -19,7 +19,18 @@ public class SA1306UnitTests : CodeFixVerifier
             await this.VerifyCSharpDiagnosticAsync(testCode, EmptyDiagnosticResults, CancellationToken.None);
         }
 
-        private async Task TestThatDiagnosticIsNotReportedImpl(string modifiers)
+        [Theory]
+        [InlineData("const")]
+        [InlineData("private const")]
+        [InlineData("internal const")]
+        [InlineData("protected const")]
+        [InlineData("protected internal const")]
+        [InlineData("internal readonly")]
+        [InlineData("protected readonly")]
+        [InlineData("protected internal readonly")]
+        [InlineData("public")]
+        [InlineData("internal")]
+        public async Task TestThatDiagnosticIsNotReported(string modifiers)
```

Comment:
Maybe not related to this pull request:
I expected to see `public const`, `public readonly`, and `protected internal` here as well, or am I missing something?